test/config-next/ra.json

```diff
@@ -3,8 +3,8 @@
     "rateLimitPoliciesFilename": "test/rate-limit-policies.yml",
     "maxConcurrentRPCServerRequests": 16,
     "maxContactsPerRegistration": 100,
-    "dnsTries": 2,
-    "debugAddr": "localhost:8002",
+    "dnsTries": 3,
+    "debugAddr": "8002",
```

Comment:
I think this config line is breaking in CI. Should probably be `:8002` - looks to be missing the `:`. Sorry I missed this before!